cli/validation/index.md

```diff
@@ -201,3 +201,18 @@ option. While convenient this is not always desirable. You can disable
 validation by passing the `-o` or `--off` option. This is usually done during
 [feature-extraction](../feature-extraction), for example
 <kbd>verapdf --off --extract somefile.pdf</kbd>.
+
+<a name="extension"></a>Non pdf extension
+------------------------------------------------
+You can validate pdf files with non-pdf extension by passing the `--nonpdfext` option. For example
+<kbd>verapdf somefile --nonpdfext</kbd>.
+
+<a name="encrypted-pdf"></a>Encrypted pdf
+------------------------------------------------
+By default, verapdf trying to decrypt encrypted file with empty password. You can validate encrypted pdf  files with non-empty password by passing the `--password` option. 
```

Comment:
By default, verapdf is trying to decrypt encrypted PDF file using empty user password.